iotedgedev/cli.py

```diff
@@ -248,6 +249,38 @@ def deploy(manifest_file):
 main.add_command(deploy)
 
 
+@solution.command(
+    name="deploy-layered",
+    context_settings=CONTEXT_SETTINGS,
+    help="Create layered deployment in IoT Hub")
+@click.option("--file",
+              "-f",
+              "manifest_file",
+              default=envvars.DEPLOYMENT_CONFIG_FILE_PATH,
+              show_default=True,
+              required=False,
+              help="Specify the deployment manifest file")
+@click.option("--layered-deployment-name",
+              "-n",
+              "layered_deployment_name",
+              show_default=True,
+              required=False,
+              help="Specify the name for a layered deployment")
+@click.option("--priority",
+              "-p",
+              show_default=True,
```

Comment:
>show_default=True [](http://example.com/codeflow?start=14&length=17)

is there default value for priority?

Reply:
Nope, removed now!

Reply:
I added default, the removed it and now still forgot to remove this. Removing tomorrow 😅